pkg/integration/tests/ui/accordion.go

```diff
@@ -37,9 +37,17 @@ var Accordion = NewIntegrationTest(NewIntegrationTestArgs{
 				Contains("commit 18"),
 			).
 			// go past commit 11, then come back, so that it ends up in the centre of the viewport
-			NavigateToLine(Contains("commit 11")).
-			NavigateToLine(Contains("commit 10")).
-			NavigateToLine(Contains("commit 11")).
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectNextItem().
+			SelectPreviousItem().
```

Comment:
This is really ugly, and I wonder if there's a better way to deal with this. It is also the only test that happens to have this problem right now, but there could easily be others, for example for a `t.Views().Commits().Lines()` assertion that expects more lines than fit on the screen. This then also depends on the screen resolution, which is even worse.

I was thinking about disabling the renderOnlyVisibleLines optimization when running integration tests, but I wasn't sure that's a good idea either.

Reply:
How about a new function called NavigateDownToLine() which just calls SelectNextItem() until the selected line matches the matcher?

Reply:
That's a good suggestion, but it would be even better to add logic to NavigateToLine so that clients don't have to worry about which one to call. And it turns out that NavigateToLine already has this comment about exactly that:
```
// NOTE: this currently assumes that BufferLines returns all the lines that can be accessed.
// If this changes in future, we'll need to update this code to first attempt to find the item
// in the current page and failing that, jump to the top of the view and iterate through all of it,
// looking for the item.
```
I implemented this in a0427bb611b1 (and dropped the hack commit again), please have a look.